src/main_thread/init/multi_thread_content_initializer.ts

```diff
@@ -1787,6 +1813,28 @@ export default class MultiThreadContentInitializer extends ContentInitializer {
                 resetMediaElement(mediaElement, url);
               });
               mediaSourceStatus.setValue(MediaSourceInitializationStatus.Attached);
+
+              if (isManagedMediaSource && "disableRemotePlayback" in mediaElement) {
+                const disableRemotePlaybackPreviousValue =
+                  mediaElement.disableRemotePlayback;
+                this._currentMediaSourceCanceller.signal.register(() => {
+                  /**
+                   * This reset the disableRemotePlayback attribute to the previous value
+                   * in order to restore the <video> element has it was set by the application
+                   * before calling the RxPlayer.
+                   */
+                  if (disableRemotePlaybackPreviousValue !== undefined) {
+                    mediaElement.disableRemotePlayback =
+                      disableRemotePlaybackPreviousValue;
+                  }
+                });
+                /**
+                 * Using ManagedMediaSource needs to disableRemotePlayback or to provide
+                 * an Airplay source alternative, such as HLS.
+                 * https://github.com/w3c/media-source/issues/320
+                 */
+                mediaElement.disableRemotePlayback = true;
```

Comment:
Is it possible to just factorize all places where we doing this as it looks fairly similar?

Reply:
Done it

Reply:
I see the factorized code but why isn't it used here?

Reply:
Yes I have replaced it once it that file but it was actually used twice, thanks will update it